barretenberg_static_lib/src/composer.rs

```diff
@@ -133,7 +133,7 @@ impl StandardComposer {
         if let Some(pi) = &public_inputs {
             let mut proof_with_pi = Vec::new();
             for assignment in pi.0.iter() {
-                proof_with_pi.extend(&assignment.to_bytes());
+                proof_with_pi.extend(&assignment.to_be_bytes());
```

Comment:
This change did not break anything? If we were formatting pub inputs as little endian before and must switch to big endian I am a little confused how we were able to verify proofs successfully before.

Reply:
Good question -- so the new API is mostly cosmetic changes. So `to_bytes` was always using big endian. Its just that now, we make it explicit. 

So the only thing I did in acvm was change:

```rust
fn to_bytes(&self) -> Vec<u8>
```

to

```rust
fn to_be_bytes(&self) -> Vec<u8>
```